README.md

```diff
@@ -80,6 +80,23 @@ steps:
       cake-bootstrap: true
 ```
 
+### `script-arguments`
+
+If you need pass same additional [argument](https://cakebuild.net/docs/writing-builds/args-and-environment-vars) value to CakeBuild, you can use `script-arguments`. 
+
+When you add an Argument: eg. 
+`var assemblyVersion = Argument<string>("versionTag", "refs/tag/v0.0.0.0");`
+then your step can look like that:
+
+```yml
+steps:
+  - name: Update-Assembly-Version
+    uses: jwickowski/cake-action@v0.0.2
```

Comment:
I know we should change it to `cake-build/cake-action@v1.0.X`
Can you tell me what version should be written here?

Reply:
Referencing the major version `cake-build/cake-action@v1` is fine.

Reply:
done